### PR TITLE
Fix/Add/Improve the documentation

### PR DESCRIPTION
**Pull request by vishnoianil**
Fixed file path's
Remove the all.yml configuration override through virthost.yml, because it doesn't seems to work.
Added ovn-kubernetes related config knobs as well in the notes
Added reference setup example and related instructions
Added troubleshooting page, to maintain a list of possible issue that user might face.
Updated all.yml and explained the master/master_slave/nodes roles
Added enable_ovn_raft config knob in the all.yml

Signed-off-by: Anil Vishnoi <vishnoianil@gmail.com>

### CODE REVIEW

**Comment by vishnoianil** [comment]
@dougbtv @s1061123 PTAL, Let me know if any of this update is not technical correct. I will update accordingly.

File: playbooks/ka-init/group_vars/all.yml [hunk 1]
```diff
@@ -93,6 +99,7 @@ kubectl_proxy_port: 8088
 # ovn vars.
 # ----------------------------
 ovn_image_repo: "docker.io/ovnkube/ovn-daemonset-u:latest"
+enable_ovn_raft: True
```
Comment (s1061123) [inline playbooks/ka-init/group_vars/all.yml]: Currently default kube-ansible set up single cluster (!= HA), so I'm just wondering it works or not.

How about to make it comment?

**Review by s1061123 — COMMENTED** [review]
Thank you for your PR to improve the README/docs!
I have one comment. Please see the diff.

File: playbooks/ka-init/group_vars/all.yml (hunk 1 above)
Comment (vishnoianil) [reply playbooks/ka-init/group_vars/all.yml]: oh yeah, my bad. It should be commented out.

**Review by s1061123 — APPROVED** [review]
Looks good to me. Let me merge it! Thank you for your PR!

**Pull request merged**